tests/page-objects/common/common.po.js

```diff
@@ -158,5 +160,7 @@ module.exports = {
 
 function openSubmenu(menuName) {
   helper.findElementByTextAndClick(hamburgerMenuOptions, menuName);
-  helper.waitForAngularComplete();
+  //throwing error: `Cannot read property 'pendingRequests' of undefined` - 
+  //helper.waitForAngularComplete();
```

Comment:
@ngaruko Did you mean to leave this in as a comment? Reading the comment it seems that the `helper.waitForAngularComplete()` call doesn't work, but I assume the fact that this PR is open means the tests run fine anyway. Does this commented-out code need to stay? I'd like to remove it if we can.

Also, I believe @newtewt had looked into the `waitForAngularComplete` stuff. @newtewt can you offer insight here?

Reply:
Just left it for reference. We have had a discussion about it and concluded that it is not necessary. So I will remove that completely